### PR TITLE
feature/GP2-1240-track-export-plan-progress-new-V2

### PR DESCRIPTION
**Pull request by depsiatwal**
(Please add a sentence or two explaining the context of this PR. Reviewers can go to the ticket for detail.)
CONTEXT: This changeset adds

- Export plan page - Mark as complete
- Track export plan progress on dashboard

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1240
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [X] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [X] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

Screenshot 2021-02-01 at 15 26 01
Screenshot 2021-02-01 at 15 26 11
Screenshot 2021-02-01 at 15 26 56
Screenshot 2021-02-01 at 15 27 27

### CODE REVIEW

File: react-components/src/components/Dashboard/index.jsx [hunk 1]
```diff
@@ -19,17 +18,23 @@ export const Dashboard = ({ sections }) => {
...
             {disabled ? (
               <div
                 className="w-full link section-list__disabled section-list__link"
                 onClick={openComingSoonModal}
                 aria-hidden="true"
                 role="button"
               >
-                <div className="bg-blue-deep-20 section-list__image-container">
+                <div
+                  className={`bg-blue-deep-20 section-list__image-container`}
```
Comment (dahfool) [inline react-components/src/components/Dashboard/index.jsx:36]: Curly brackets not needed here, bg-blue-deep-20 section-list__image-container is a `string`

File: react-components/src/components/SectionComplete/SectionComplete.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          id="checkbox_complete"
+          onChange={toggleComplete}
+          checked={isComplete ? true : false}
+        />
+        <label htmlFor="checkbox_complete">Yes</label>
+      </div>
+    </>
+  )
+}
+SectionComplete.propTypes = {
```
Comment (dahfool) [inline react-components/src/components/SectionComplete/SectionComplete.jsx:43]: PropsTypes which are not required, need a default value. Please check the console for warnings.

File: react-components/src/components/Dashboard/index.jsx (hunk 1 above)
Comment (jakehopking) [reply react-components/src/components/Dashboard/index.jsx]: Good spot.

File: react-components/src/components/SectionComplete/SectionComplete.jsx (hunk 2 above)
Comment (jakehopking) [reply react-components/src/components/SectionComplete/SectionComplete.jsx:43]: Updated.

File: react-components/src/components/SectionComplete/SectionComplete.jsx [hunk 3]
```diff
@@ -44,8 +44,8 @@ SectionComplete.propTypes = {
   current_section: PropTypes.shape({
     is_complete: PropTypes.bool.isRequired,
     url: PropTypes.string.isRequired,
-    title: PropTypes.string,
+    title: PropTypes.string.isRequired,
     lessons: PropTypes.arrayOf(PropTypes.string),
-    disabled: PropTypes.bool,
+    disabled: PropTypes.bool.isRequired,
   }),
```
Comment (dahfool) [inline react-components/src/components/SectionComplete/SectionComplete.jsx:50]: You can .isRequired to shape like PropTypes.shape({}).isRequired.

**Comment by dahfool** [comment]
@jakehopking The code coverage for FE has started to drop, we need some unit tests.

File: react-components/src/components/SectionComplete/SectionComplete.jsx (hunk 3 above)
Comment (jakehopking) [reply react-components/src/components/SectionComplete/SectionComplete.jsx:50]: Thanks :)

**Review by jakehopking — APPROVED** [review]

**Pull request merged**